### PR TITLE
Published clusters configuration

### LINKED ISSUE #184: Support for getting published cluster configurations

### PR DESCRIPTION
**Pull request by juliamcclellan**
Fix #184

### CODE REVIEW

File: tests/unit/test_published_clusters.py [hunk 1]
```diff
@@ -17,15 +58,15 @@ def test_published_clusters():
...
+    _unified_dataset_json = {
         "id": "unify://unified-data/v1/datasets/8",
         "name": "Project_1_unified_dataset",
         "version": "10",
         "relativeId": "datasets/8",
         "externalId": "Project_1_unified_dataset",
     }
 
-    published_clusters_json = {
+    _published_clusters_json = {
```
Comment (charlottemoremen) [inline tests/unit/test_published_clusters.py]: why the underscore before "published"?

Comment (juliamcclellan) [reply tests/unit/test_published_clusters.py]: A lot of the existing tests do that for internal json variables

Comment (nbateshaus) [reply tests/unit/test_published_clusters.py]: The leading underscore is meaningful: 
https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables
"Use one leading underscore only for non-public methods and instance variables."

File: tamr_unify_client/models/project/mastering.py [hunk 2]
```diff
@@ -85,6 +85,20 @@ def published_clusters(self):
...
+
+        :returns: The returned JSON body, as specified in the
+            `Public Docs for Cluster Configurations
+            <https://docs.tamr.com/reference#the-published-clusters-configuration-object>`_.
+        :rtype: dict
+        """
+        return (
+            self.client.get(self.api_path + "/publishedClustersConfiguration")
+            .successful()
+            .json()
```
Comment (nbateshaus) [inline tamr_unify_client/models/project/mastering.py]: Even though it's a stupid class with only one member, we should provide a Python class for this.

Comment (juliamcclellan) [reply tamr_unify_client/models/project/mastering.py]: Sad

File: tamr_unify_client/models/project/cluster_configuration.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    def from_json(cls, client, data, api_path):
+        return super().from_data(client, data, api_path)
+
+    @property
+    def relative_id(self):
+        """:type: str"""
+        return self.api_path
+
+    @property
+    def time_to_live(self):
```
Comment (nbateshaus) [inline tamr_unify_client/models/project/cluster_configuration.py]: Should be `versions_time_to_live`

File: tamr_unify_client/models/project/cluster_configuration.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+class PublishedClustersConfiguration(BaseResource):
+    @classmethod
+    def from_json(cls, client, data, api_path):
+        return super().from_data(client, data, api_path)
+
+    @property
+    def relative_id(self):
+        """:type: str"""
+        return self.api_path
```
Comment (nbateshaus) [inline tamr_unify_client/models/project/cluster_configuration.py]: This needs an explanatory comment like the one in `attribute/resource.py`

**Pull request merged**